src/main/java/mate/academy/internetshop/controllers/UserDeleteController.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.internetshop.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.internetshop.lib.Injector;
+import mate.academy.internetshop.service.UserService;
+
+public class UserDeleteController extends HttpServlet {
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy.internetshop");
+    private final UserService userService = (UserService) INJECTOR.getInstance(UserService.class);
```

Comment:
why in some controllers you have made services private final and just private in others?